B827EBFFFE60D602.json

```diff
@@ -0,0 +1,17 @@
+{
+  " gateway_conf " : {
+    " gateway_ID " : " B827EBFFFE60D602 " ,
```

Comment:
Please remove the extra leading and trailing whitespaces before & after the EUI.